README.md

```diff
@@ -1,41 +1,41 @@
-# Astronomer Deploy Action
-Custom Github Action to create CI/CD pipelines that deploys Airflow code to Astro Airflow Deployments during a CI/CD workflow. See [Astronomer's CI/CD documentation](https://docs.astronomer.io/astro/ci-cd) page for more information on creating CI/CD pipelines for Astro.
+# Deploy Action
```

Comment:
```suggestion
# Deploy to Astro
```
Looking at the [GitHub Marketplace](https://github.com/marketplace?category=deployment&query=sort%3Apopularity-desc&type=actions&verification=), this seems to be the most common naming format for Deployment actions.